src/pow.cpp

```diff
@@ -147,10 +218,15 @@ unsigned int GetNextWorkRequired(const CBlockIndex* pindexLast, const CBlockHead
         return pindexLast->nBits;
     }
 
-    return CalculateNextWorkRequired(pindexLast, params);
+    return CalculateNextWorkRequired(pindexLast, params, protocol_cleanup);
 }
 
-unsigned int CalculateNextWorkRequired(const CBlockIndex* pindexLast, const Consensus::Params& params)
+uint32_t GetNextWorkRequiredAux(const CBlockIndex* pindexLast, const CBlockHeader& block, const Consensus::Params& params)
+{
+    return CalculateNextWorkRequiredAux(pindexLast, params);
+}
+
+unsigned int CalculateNextWorkRequired(const CBlockIndex* pindexLast, const Consensus::Params& params, bool protocol_cleanup)
```

Comment:
As a general suggestion, I think it would be helpful to split the aux and non-aux versions cleanly into two parts of the file, instead of them being intermingled throughout the file. (That, or put them cleanly paired together.)

Reply:
I moved the aux versions to the end of the file, so they're all grouped together.